source/config/config_test.go

```diff
@@ -89,3 +91,36 @@ func TestParseGlobalConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestGetOrderedFields(t *testing.T) {
+	var configTests = []struct {
+		name     string
+		input    []string
+		expected []string
+	}{
+		{
+			name:     "Nil Input",
+			input:    nil,
+			expected: []string{"id", "createdAt", "updatedAt", "firstName", "lastName", "email"},
+		},
+		{
+			name:     "Empty Input",
+			input:    []string{},
+			expected: []string{"id", "createdAt", "updatedAt", "firstName", "lastName", "email"},
+		},
+		{
+			name:     "With fields",
+			input:    []string{"email", "firstName"},
+			expected: []string{"id", "createdAt", "updatedAt", "email", "firstName"},
+		},
+	}
+
+	for _, tt := range configTests {
+		t.Run(tt.name, func(t *testing.T) {
+			actual := getOrderedFields(tt.input)
```

Comment:
It feels a bit wrong to test unexported functions directly. Why not provide a config map with a list of fields, and expect a config struct with ordered fields?

Reply:
removed and as suggested it is added in config map, also updated in Readme as Note

Reply:
We are still using the unexported functio, instead of the Parse() method.

Reply:
code now is updated. pls check once again